org-roam-graph.el

```diff
@@ -250,7 +250,9 @@ If MAX-DISTANCE is non-nil, only nodes within the given number of steps are show
                     (list file)))
```

Comment:
Note that the filter is only taken into account /after/ computing the connected component. In the worst case this might lead to unexpected behavior by including too many nodes, if two clusters are only connected via a single node. E.g.
```
cluster A -> node B -> cluster C
```
Would lead to nodes from both clusters being displayed, while only excluding node B.